lib/core/network/api/aws_amplify/amplify_service.dart

```diff
@@ -98,7 +98,7 @@ class AmplifyService {
     cognitoUser = null;
   }
 
```

Comment:
I renamed the method because we have another "loginUser" method and that's very confusing.